PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobLocalRunConfigurationProducer.java

```diff
@@ -91,59 +100,66 @@ private void setupConfiguration(LivySparkBatchJobRunConfiguration configuration,
         setupConfigurationModule(context, configuration);
     }
 
-    private static Optional<String> getNormalizedClassName(@NotNull PsiClass clazz) {
+    private static Optional<String> getNormalizedClassName(@NotNull final PsiClass clazz) {
         return Optional.ofNullable(SparkContextUtilsKt.getNormalizedClassNameForSpark(clazz));
     }
 
     /**
      * The function to help reuse RunConfiguration
+     *
      * @param jobConfiguration Run Configuration to test
-     * @param context current Context
+     * @param context          current Context
```

Comment:
No need for these spaces

Reply:
This change happened after I do auto-format in IntelliJ. The space can make the description of the parameter aligned vertically.
![image](https://user-images.githubusercontent.com/32627233/79180058-90755400-7e3b-11ea-871d-4fb67fb4a10f.png)